packages/portal-proto/src/features/set-operations/CountButtonWrapperForSetsAndCases.tsx

```diff
@@ -137,6 +137,8 @@ const CountButtonWrapperForSetsAndCases: React.FC<
     return await createSet({
       // TODO: possibly add error handling
       filters: filters,
+      intent: "user",
+      set_type: "frozen",
```

Comment:
<img width="1021" alt="Screenshot 2024-04-05 at 2 34 47 PM" src="https://github.com/NCI-GDC/gdc-frontend-framework/assets/4624053/b3c1841a-fc83-40b1-a25b-a83fbd0aa42c">

Reply:
i don't think we want 'frozen' for genes/ssms sets -- those should be 'mutable'.

i can see reasons for setting case sets as 'frozen'.

is it possible to distinguish, e.g.
- genes/ssms sets: user, mutable
- case sets/cohorts: portal, frozen